src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/TseViewApplicationHelper.java

```diff
@@ -136,6 +140,21 @@ public static void setDataForTseApplicationSummaryAndResponses(CaseData caseData
         caseData.setTseApplicationSummaryAndResponsesMarkup(applicationSummary + respondTablesCollection);
     }
 
+    /**
+     * Checks if application is viewable for respondent.
+     *
+     * @param genericTseApplicationTypeItem - generic application type
+     */
+    public static boolean applicationsSharedWithRespondent(
+            GenericTseApplicationTypeItem genericTseApplicationTypeItem) {
```

Comment:
nitpick: could the name of this param be shortened so it can fit on a single line?